src/index.html

```diff
@@ -3,12 +3,59 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="ul">
+          <li class="li">
+            <a 
+              href="#apple"
+              class="a is-active"
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a 
              href="#apple"
              class="nav__link is-active"
```
Погана ідея - давати назви класам такі ж, як назви тегів (для nav це не критично, адже навігація, завзичай, одна на сторінці)